docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PG_VERSION_MAJOR=16
```

Comment:
We may want to still keep 16 as the default, so it can be built locally as well

Reply:
Dockerfile is a bit confusing here - not setting a default value will make it an empty string. Or maybe my mistake. I'm removing this commit.

Reply:
Maybe we can add a comment in the Dockerfile to clarify that the default value gets updated in the publish workflow?